src/app/homepage/Styles.tsx

```diff
@@ -1,12 +1,13 @@
 'use client'
-import { Color, FontVariant } from '@/app/theme'
+import { Color, FontVariant, ScreenSize } from '@/app/theme'
 import styled from '@emotion/styled'
 
 export const Section = styled.section<{ altBackground?: boolean }>`
   background-color: ${props => (props.altBackground ? '#F6F6F6' : 'white')};
   margin: 0 auto;
   width: 100%;
-  padding: 48px 96px;
+  padding: clamp(48px, calc(48px + 0.113 * (100vw - ${ScreenSize.md})), 48px)
```

Comment:
It will be good to define 0.113 as a constant for later code management. Explain where the number comes from when naming the constant

Reply:
https://github.com/kixlab/website/pull/21/commits/dab61e3447af4ebe8fa13929655834940dec436e